src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
 public class WorkWithFile {
+    static final String SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(makeReport(readFromFile(fromFileName)),toFileName);
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        ArrayList<String> list = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String value;
+            while ((value = bufferedReader.readLine()) != null) {
+                list.add(value);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Cant read from file" + fromFileName, e);
+        }
+        return list.toArray(new String[0]);
+    }
+
+    private String makeReport(String[] arrayFromFile) {
+        int supply = 0;
+        int buy = 0;
+        for (String s : arrayFromFile) {
+            String[] split = s.split(SEPARATOR);
+            if (split[0].startsWith("s")) {
+                supply += Integer.parseInt(split[1]);
+            } else {
+                buy += Integer.parseInt(split[1]);
```

Comment:
avoid magic numbers. 

all indexes should be constant with self-descriptive name.